src/Core/ClassInfo.php

```diff
@@ -560,4 +560,41 @@ public static function parse_class_spec($classSpec)
         static::$_cache_parse[$classSpec] = $result;
         return $result;
     }
+
+
+    /**
+     * Returns a list of classes with a particular extension applied
+     *
+     * This reflects all extensions added (or removed) both via yaml as well as dynamically
```

Comment:
```suggestion
     * This reflects all extensions added (or removed) both via the configuration API as well as dynamically
```

Includes PHP private statics and YAML config